website/client/assets/scss/item.scss

```diff
@@ -19,6 +19,10 @@
   margin-bottom: 8px;
 }
 
+.pet-slot {
```

Comment:
just one small change and this should be good to go, can you change the selector from `.pet-slot` to `.item.pet-slot` so it's more specific?

Reply:
done